### PR TITLE
Add PYACTION / UDQ testcase

### PR DESCRIPTION
**Pull request by joakim-hove**
This is identical to the existing `UDQ_WCONPROD` but the UDQ values are assigned through Python code instead of through expressions parsed in the .DATA file

In addition to this test it would be advantageous to have a test like 
```
results(UDQ_WCONPROD) == results(PYACTION_WCONPROD)
```

~~opm-test: https://github.com/OPM/opm-tests/pull/283~~
opm-common: https://github.com/OPM/opm-common/pull/1620

### CODE REVIEW

**Comment by akva2** [comment]
missing update to update_reference_data.sh

File: compareECLFiles.cmake [hunk 1]
```diff
@@ -399,6 +399,13 @@ add_test_compareECLFiles(CASENAME udq_wconprod
                          REL_TOL ${rel_tol}
                          DIR udq_actionx)
 
+add_test_compareECLFiles(CASENAME udq_pyaction
```
Comment (akva2) [inline compareECLFiles.cmake:402]: this needs to be wrapped in if(OPM_ENABLE_PYTHON)

Comment (joakim-hove) [reply compareECLFiles.cmake]: Done

Comment (joakim-hove) [reply compareECLFiles.cmake]: The relevant cmake flag - `OPM_ENABLE_EMBEDDED_PYTHON` is defined and set in opm-common; seems it is not available here? The test is never run.

Comment (akva2) [reply compareECLFiles.cmake]: right. in that case it needs to be published through the opm-common-config.cmake.

Comment (joakim-hove) [reply compareECLFiles.cmake]: Ehh - exactly which file is this?

**Pull request merged**